model/perf_test.go

```diff
@@ -501,6 +501,11 @@ func getTestPerformanceResults() (*PerformanceResult, *PerformanceResult) {
 			Tags:      []string{"tag1", "tag2", "tag3"},
 			Arguments: map[string]int32{"threads": 64},
 			Mainline:  true,
+			OverrideInfo: OverrideInfo{
+				OverrideMainline: true,
+				Reason:           "test_reason",
+				User:             "test_user",
+			},
```

Comment:
I've added the OverrideInfo to the test data here and as expected, this does not break anything.